modules/log6x-logging-loki-gui-install.adoc

```diff
@@ -0,0 +1,137 @@
+[id="log6x-logging-loki-gui-install_{context}"]
+= Installing {logging-uc} by using the web console
+
+To install and configure logging on your {product-title} cluster, you must first install {loki-op}, which manages the log store. This can be done from the OperatorHub within the web console.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+* You have access to a supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, {rh-storage}).
+
+.Procedure
+
+. In the {product-title} web console *Administrator* perspective, go to *Operators* -> *OperatorHub*.
+
+. Type {loki-op} in the *Filter by keyword* field. Click *{loki-op}* in the list of available Operators, and then click *Install*.
++
+[IMPORTANT]
+====
+The Community {loki-op} is not supported by Red{nbsp}Hat.
+====
+
+. Select *stable-x.y* as the *Update channel*.
++
+The {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it will be created for you.
+
+. Select *Enable Operator-recommended cluster monitoring on this namespace.*
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+. For *Update approval* select *Automatic*, then click *Install*.
++
+If the approval strategy in the subscription is set to *Automatic*, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to *Manual*, you must manually approve pending updates.
++
+[NOTE]
+====
+An Operator might display a `Failed` status before the installation finishes. If the Operator install completes with an `InstallSucceeded` message, refresh the page.
+====
+
+. While the operator installs, you can create the namespace where the log store will be deployed to.
+
+.. Click *+* in the top right of the screen to access the *Import YAML* page. 
+
+.. Add the YAML definition for the `openshift-logging` namespace:
++
+.Example `namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging # <1>
+  labels:
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> `openshift-logging` is the namespace dedicated for all {logging} workloads.
+<2> A string value that specifies the label, as shown, to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+.. Click *Create*.
+
+. Create a secret with the credentials to access the object storage.
+
+.. Click *+* in the top right of the screen to access the *Import YAML* page. 
+
+.. Add the YAML definition for the secret. For example, create a secret to access Amazon Web Services (AWS) s3:
++
+.Example `Secret` object
```

Comment:
@theashiot  Is there any plan to deploy lokistack on COO managed cluster?

Reply:
You mean CCO? I believe @theashiot was planning to write those sections in the `Loki object storage` section once he creates the page about the `Log Store`. If you actually meant COO in one of the discussions of this PR we talked about adding sections for that on a follow up.

For context @theashiot CCO stands for cloud-credentials-operator & COO cluster-observabilty-operator

Reply:
Thanks, @JoaoBraveCoding! Yes COO will be added as a follow-up, if i haven't created a Jira for that yet, i'll create one and add here.  About CCO, I agree with what Joao says.

Reply:
Created the Jira: https://issues.redhat.com/browse/OBSDOCS-1749